### PR TITLE
Statistics element

### PR DESCRIPTION
**Pull request by DKunin**
Added statistics element and examples.

Had to hack logic of children `ui` class, when `sui-statistic` is inside of group `sui-statistics`. It would probably be helpful to create global helper, to determine if the component is inside a group or not, since some other components have similar logic, Card for example.

### CODE REVIEW

File: src/views/Statistic/Statistic.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    size: Enum.Size,
+    floated: Enum(['left', 'right']),
+    inverted: {
+      type: Boolean,
+      description: 'Should the colors be inverted',
+    },
+    inGroup: {
+      type: Boolean,
+      description: 'For now manual override, when used inside Statistic Group, to avoid adding extra ui class',
+    },
```
Comment (mariolamacchia) [inline src/views/Statistic/Statistic.jsx]: I get what you wanted to achieve here, but the additional `ui` class would not break anything.
In fact, in your examples you're using camel case for props (it should be `in-group`, not `inGroup`) so you aren't using this one at all. I'd say to remove it completely

Comment (DKunin) [reply src/views/Statistic/Statistic.jsx]: For this particular example - it seems to me, that it does have an affect. 
Pics for proof of how blocks behave with or without `.ui > .ui`:
https://i.imgur.com/AmlqDpo.png
https://i.imgur.com/4rQ78Mx.png

Should I still remove the inGroup logic?

As for camelCase - I think vue magically does some converting under the hood, so camelCase and `in-group` will both work:
https://i.imgur.com/sVX07CV.png

Comment (mariolamacchia) [reply src/views/Statistic/Statistic.jsx]: You're right, it is making a small difference. Let's leave it like this for now, I'll come up with a solution later on.

Comment (DKunin) [reply src/views/Statistic/Statistic.jsx]: Great! Thanks!

**Pull request merged**